README.md

```diff
@@ -1,14 +1,12 @@
 # Sending Data to Our Service
 
-If you use OpenTelemetry, you can configure the application to send traces or metrics to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) using the Wavefront proxy or the OpenTelemetry Collector.
-
-Send trace data or metrics data to our service using only the Wavefront proxy:
+If you use OpenTelemetry, you can configure the application to send traces or metrics to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) using the Wavefront proxy.
 
 <img src="images/opentelemetry_proxy_tracing.png" alt="A data flow diagram that shows how the data flows from your application to the proxy, and then to our service" style="width:750px;"/>
 
 Follow these steps:
 
-1. [Install the Wavefront proxy](https://docs.wavefront.com/proxies_installing.html) version 11.3 or higher.
+1. [Install the Wavefront proxy](https://docs.wavefront.com/proxies_installing.html).
```

Comment:
In my humble opinion, we should leave 11.3 or higher in. 11.2 has bugs in it so that the OTel stuff doesn't work.